src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -139,6 +139,29 @@
          [Spinner {:text (:banner props)}]]]))})
 
 
+(react/defc SafeBlocker
```

Comment:
I'm not sure I see how this is "safer" than the other blocker. Maybe `DelayedBlocker`? Also, is there a way this could share more code with `Blocker` or even the new modal (or could the modal be modified to include this use case)?